shell.nix

```diff
@@ -22,10 +22,6 @@ in {
     nativeBuildInputs = [
       specs.agda
       cabal-install
-      (haskellPackages.ghcWithPackages (pkgs: with pkgs; [
-        specs.ledger.hsExe
-        specs.midnight.hsExe
-      ]))
```

Comment:
We don't need this anymore?

Reply:
I don't think it makes sense to build the project when entering `nix-shell`